content/es/pod.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Pod
+status: Feedback Appreciated
+category: Concepto
+tags: ["infraestructura", "fundamento"]
+---
+
+En [Kubernetes](/kubernetes/), un pod es la unidad más básica dentro de un clúster. Un pod puede contener uno o más [contenedores](/container/), que comparten el mismo espacio de red y de almacenamiento.
+
+Kubernetes gestiona los pods, garantizando su alta disponibilidad y escalabilidad mediante escalamiento [vertical](/vertical-scaling/) o [horizontal](/horizontal-scaling/) según sea necesario.
+
+## Problema que aborda
+
+Por lo general, los contenedores funcionan de forma independiente, ejecutando tareas específicas por separado. Sin embargo, en ocasiones, es necesario que interactúen y sean gestionados en conjunto ya que si se hiciera de forma individual conllevaría a la redundancia en tareas de gestión.
```

Comment:
This paragraph localized seems to be different from the [original one](https://github.com/cncf/glossary/blob/main/content/en/pod.md?plain=1#L15-L20):

```
While containers generally act as independent units that run and control a particular workload, 
there are cases when containers need to interact and be controlled in a tightly coupled manner. 

If each of these closely related containers were managed individually, it would lead to redundant management tasks.
For example, the operator would have to repeatedly determine the placement of related containers to ensure they remain together.
And although the lifecycles of these related containers need to be synchronized, they can only be managed individually.
```